akka-projection-cassandra/src/main/scala/akka/projection/cassandra/internal/CassandraProjectionImpl.scala

```diff
@@ -7,10 +7,10 @@ package akka.projection.cassandra.internal
 import java.util.concurrent.CompletionStage
 
 import scala.collection.immutable
+import scala.compat.java8.DurationConverters._
```

Comment:
I don't think this exists in the compat for 2.12 (we have to use same version as Akka). We can use `akka.util.JavaDurationConverters` instead.

Reply:
Ok, I will change